Packs/IronPort/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Cisco Email Security Appliance (IronPort)",
     "description": "Cisco Email Security protects against ransomware, business email compromise, spoofing, and phishing",
     "support": "xsoar",
-    "currentVersion": "2.0.25",
+    "currentVersion": "2.0.26",
```

```suggestion
    "currentVersion": "2.1.0",
```
Don't forget to change the RN file name as well